src/components/facet-browser/helper.ts

```diff
@@ -100,4 +102,15 @@ export function getAllFilterPathsAsString(filterObject: {
   return allFilterPathsAsString;
 }
 
+export const getFirstMaterialTypeFromFilters = (
```

Comment:
This helper is placed in `src/components/facet-browser` but is used in `src/components/search-result-list[...]`.

Please move the component to an appropriate helper.